_translator-files/po/fr/Directories.po

```diff
@@ -53,8 +53,6 @@ msgstr "Administrer un annuaire"
 
 #. type: Plain text
 #: ../wiki/en/Directories.md:14
-#, fuzzy
-#| msgid "This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-modes)."
 msgid "This is a specialised Jamulus Server configuration, as described in [Server Modes](Running-a-Server#server-modes)."
 msgstr "Il s'agit d'une configuration spécialisée du serveur Jamulus, comme décrit dans [Types de serveurs](Running-a-Server#types-de-serveur)."
```

Comment:
This link is wrong.

Reply:
Same here : https://jamulus.io/fr/wiki/Running-a-Server#types-de-serveur

Reply:
As above. What used to be "Server types" is now "Server modes".

Reply:
fixed too as well (I think)